sensirion_common.c

```diff
@@ -63,6 +63,11 @@ int8_t sensirion_common_check_crc(uint8_t *data, uint16_t count,
     return STATUS_OK;
 }
 
+int16_t sensirion_i2c_general_call_reset(void) {
+    const uint8_t data = 0x06;
```

Comment:
How about making it an array, such that the size parameter is coupled with the data
```
const uint8_t data[] = {0x06};
return sensirion_i2c_write(0, data, sizeof(data));
```

Reply:
`sizeof()` might need an explicit cast to whatever `sensirion_i2c_write` takes to avoid warnings on some platforms.

Reply:
Fixed. `sizeof()` also works without an array.